.github/workflows/fly.yaml

```
@@ -0,0 +1,58 @@
+name: Fly Preview
+
+on:
+  pull_request:
+
+jobs:
+  preview:
+    runs-on: ubuntu-latest
+    env:
+      FLY_API_TOKEN: ${{ secrets.FLY_API_TOKEN }}
+      FLY_ORGANIZATION_SLUG: supabase-dev
```

```suggestion
      FLY_ORGANIZATION_SLUG: ${{ secrets.FLY_ORG_SLUG }}
```

I suggest we allow the user to byo org